### PR TITLE
Restructure the Кubernetes section

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-761.surge.sh

File: modules/ROOT/pages/kubernetes/authentication-authorization.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
+[[configure-authentication-and-authorization]]
+= Authentication and authorization
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/authentication-authorization.adoc:2]: might want to replace authorization with Authorization

Comment (renetapopova) [reply modules/ROOT/pages/kubernetes/authentication-authorization.adoc:2]: No, we use sentence case for titles. If you've seen anywhere capital case, it's a mistake.

File: modules/ROOT/pages/kubernetes/accessing-neo4j.adoc [hunk 2]
```diff
@@ -198,29 +363,43 @@ Some of them can be customized by adding extra configuration to the helm deploym
                                                .1+| `adminService.annotations`     .1+| Annotations object for headless (DNS) service.
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/accessing-neo4j.adoc:363]: there are no separate fields for standalone and cluster in values.yaml

there is no such key as neo4jService,adminService,externalService

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/kubernetes/accessing-neo4j.adoc (hunk 2 above)
Comment (harshitsinghvi22) [reply modules/ROOT/pages/kubernetes/accessing-neo4j.adoc:363]: replace  the following in standalone 
neo4jService.annotations --> services.default.annotations , 
adminService.annotations --> services.admin.annotations , 
externalService.annotations --> services.neo4j.annotations 

in cluster replace
services.neo4j.annotations  --> services.default.annotations 
annotations --> services.neo4j.annotations

**Review by harshitsinghvi22 — APPROVED** [review]

**Pull request merged**